plugin/pom.xml

```diff
@@ -125,6 +135,13 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
         <version>1.9</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-client</artifactId>
```

Comment:
The invoker-plugin executes the `verify.groovy` scripts which don't seem to inherit the dependencies in the pom. I had to add it here to be able to use the `DefaultDockerClient` in the test.